Python/Product/Core/Core.csproj

```diff
@@ -101,6 +101,10 @@
       <Project>{DECC7971-FA58-4DB0-9561-BFFADD393BBD}</Project>
       <Name>Microsoft.PythonTools.Debugger</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Debugger.Concord\Debugger.Concord.csproj">
+      <Project>{515F701E-336D-4D58-AF4B-E976BC33C957}</Project>
+      <Name>Microsoft.PythonTools.Debugger.Concord</Name>
+    </ProjectReference>
```

Comment:
This should be unnecessary since there is a reference via PythonTools.csproj. Or you can remove the PythonTools.csproj reference if it isn't needed in that assembly.